src/test/groovy/pl/exsio/querydsl/entityql/groovy/native_sql/it/BaseClassForDynamicTests.groovy

```diff
@@ -0,0 +1,59 @@
+package pl.exsio.querydsl.entityql.groovy.native_sql.it
+
+import org.springframework.beans.factory.annotation.Autowired
+import pl.exsio.querydsl.entityql.entity.scanner.nativesql.Table
+import schemacrawler.schema.Catalog
+import schemacrawler.schema.Column
+import schemacrawler.schema.Schema
+import schemacrawler.schemacrawler.SchemaCrawlerOptions
+import schemacrawler.schemacrawler.SchemaCrawlerOptionsBuilder
+import schemacrawler.utility.SchemaCrawlerUtility
+import spock.lang.Specification
+
+import javax.sql.DataSource
+
+abstract class BaseClassForDynamicTests extends Specification {
```

Comment:
please rename to DynamicRuntimeTestsBase

Reply:
correction, rename to RuntimeDBTestBase

Reply:
renamed to RuntimeDBTestBase